projects/plugins/crm/modules/woo-sync/admin/settings/main.page.php

```diff
@@ -56,10 +56,11 @@ function jpcrm_settings_page_html_woosync_main() {
 		$updatedSettings['wctagproductprefix'] = ! empty( $_POST['wctagproductprefix'] ) ? zeroBSCRM_textProcess( $_POST['wctagproductprefix'] ) : '';
 
 		// switches
-		$updatedSettings['wcinv']  = ! empty( $_POST['wpzbscrm_wcinv'] );
-		$updatedSettings['wcprod'] = ! empty( $_POST['wpzbscrm_wcprod'] );
-		$updatedSettings['wcport'] = ! empty( $_POST['wpzbscrm_wcport'] ) ? preg_replace( '/\s*,\s*/', ',', sanitize_text_field( $_POST['wpzbscrm_wcport'] ) ) : '';
-		$updatedSettings['wcacc']  = ! empty( $_POST['wpzbscrm_wcacc'] );
+		$updatedSettings['wcinv']             = ! empty( $_POST['wpzbscrm_wcinv'] ); // phpcs:ignore WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase, WordPress.Security.NonceVerification.Missing
+		$updatedSettings['wccrminvreference'] = ! empty( $_POST['wpzbscrm_wccrminvreference'] ); // phpcs:ignore WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase, WordPress.Security.NonceVerification.Missing
+		$updatedSettings['wcprod']            = ! empty( $_POST['wpzbscrm_wcprod'] ); // phpcs:ignore WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase, WordPress.Security.NonceVerification.Missing
+		$updatedSettings['wcport']            = ! empty( $_POST['wpzbscrm_wcport'] ) ? preg_replace( '/\s*,\s*/', ',', sanitize_text_field( $_POST['wpzbscrm_wcport'] ) ) : ''; // phpcs:ignore WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase, WordPress.Security.NonceVerification.Missing, WordPress.Security.ValidatedSanitizedInput.MissingUnslash
+		$updatedSettings['wcacc']             = ! empty( $_POST['wpzbscrm_wcacc'] ); // phpcs:ignore WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase, WordPress.Security.NonceVerification.Missing
```

Comment:
```suggestion
		// phpcs:disable WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase, WordPress.Security.NonceVerification.Missing
		$updatedSettings['wcinv']             = ! empty( $_POST['wpzbscrm_wcinv'] );
		$updatedSettings['wccrminvreference'] = ! empty( $_POST['wpzbscrm_wccrminvreference'] );
		$updatedSettings['wcprod']            = ! empty( $_POST['wpzbscrm_wcprod'] );
		$updatedSettings['wcport']            = ! empty( $_POST['wpzbscrm_wcport'] ) ? preg_replace( '/\s*,\s*/', ',', sanitize_text_field( wp_unslash( $_POST['wpzbscrm_wcport'] ) ) ) : ''; // phpcs:ignore WordPress.Security.ValidatedSanitizedInput.MissingUnslash - wcport values can be saved with slashes.
		$updatedSettings['wcacc']             = ! empty( $_POST['wpzbscrm_wcacc'] );
		 // phpcs:enable WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase, WordPress.Security.NonceVerification.Missing
```
Perhaps this is slightly cleaner? With the MissingUnslash sniff, I believe `wcport` is related to the list of Jetpack CRM contact fields that can be edited via WooCommerce My Account. I added one with a slash and it does save so it does look like the sniff needs to stay - maybe with an additional note.